bitnami/redis/templates/redis-node-statefulset.yaml

```diff
@@ -341,7 +341,38 @@ spec:
             - name: metrics
               containerPort: 9121
           resources: {{- toYaml .Values.metrics.resources | nindent 12 }}
-  {{- end }}
+        {{- end }}
+        {{- if .Values.sentinel.metrics.enabled }}
+        - name: sentinel-metrics
+          image: {{ template "sentinel.metrics.image" . }}
+          imagePullPolicy: {{ .Values.sentinel.metrics.image.pullPolicy | quote }}
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          command:
+            - /go/bin/redis_sentinel_exporter{{- range $key, $value := .Values.metrics.extraArgs }} --{{ $key }}={{ $value }}{{- end }}
```

Comment:
Does this work with the bitnami image?

Reply:
Updated binary path to work with bitnami image.